predictionnet/utils/huggingface.py

```diff
@@ -0,0 +1,66 @@
+import os
+from typing import List
+
+from huggingface_hub import HfApi, model_info
+
+from predictionnet.protocol import Challenge
+
+
+class HF_interface:
+    def __init__(self):
+        token = os.getenv("HF_ACCESS_TOKEN")
+        if token is None:
+            raise ValueError(
+                "Huggingface access token not found in environment variables, set it as 'HF_ACCESS_TOKEN'."
+            )
+        collection_slug = os.getenv("HF_COLLECTION_SLUG")
+        if collection_slug is None:
+            raise ValueError(
+                "Huggingface collection slug not found in environment variables, set it as 'HF_COLLECTION_SLUG'."
+            )
+        self.api = HfApi(token=token)
+        self.collection_slug = collection_slug
+        self.collection = self.get_models()
+
+    def get_models(self):
+        collection = self.api.get_collection(collection_slug=self.collection_slug)
+        return collection
+
+    def add_model_to_collection(self, repo_id, model_id) -> None:
```

Comment:
Would it make sense to return a boolean, instead of `None` to indicate if the model was successfully added? Something like this could help with logging down the line

Reply:
@hscott-fd

Reply:
Fixed